clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts

```diff
@@ -124,10 +113,20 @@ describe.skip("Config wizard with plus settings", () => {
         const { body } = interception.request;
         expect(body.map((s) => s.fides_key)).to.eql(systems);
       });
+
+      // Make sure there are only two systems in this table
+      cy.getByTestId("systems-classify-table");
+      cy.getByTestId("status-vzmgr-service");
+      cy.getByTestId("status-kube-dns");
+      cy.get("table")
+        .find("tr")
+        .then((rows) => {
+          expect(rows.length).to.eql(3); // +1 for the header row
```

Comment:
`find("tbody > tr")` could spare us the math.

Reply:
oh nice tip, I'm all about sparing myself math